utils/keycodes.js

```diff
@@ -1,3 +1,10 @@
+/**
+ * Browser dependencies
+ */
+const { userAgent } = window.navigator;
```

Comment:
Is the destructuring worthwhile if we're using it in one spot?

Reply:
It's a bit strange indeed if the use immediately follows, but I find it clear to state browser dependencies.